File: beginner/java/2147.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        short casos;
        String palavra;
        float resultado = 0.01f;
    
    
        casos = scanner.nextShort();
        scanner.nextLine();
    
        while (casos!=0){
    
            palavra = scanner.nextLine();
    
            System.out.printf("%.2f%n", (palavra.length())*resultado);
    
            casos--;
        }
        

        scanner.close();
    }
}

File: beginner/c/2147.c
#include <stdio.h>
#include <string.h>

int main() {
    unsigned short casos;
	char palavra[10001];
	float resultado = 0.01f;


	scanf("%hu", &casos);

	while (casos--){

		scanf(" %s", palavra);

		printf("%.2f\n", strlen(palavra)*resultado);


	}

    return 0;
}
